scripts/main.js

```diff
@@ -4,35 +4,37 @@ var orders = [];
 var url = 'https://dc-coffeerun.herokuapp.com/api/coffeeorders';
 
 var retrieveOrders = function () {
-    orders = []
-    $.ajax(url, {
-        success: function (data) {
+    var retrieveOrdersPromise = fetch(url);
+    retrieveOrdersPromise.then(function (data) {
+        orders = []
+        var ordersToJson = data.json();
+        ordersToJson.then(function (data) {
             for (key in data) {
```

Comment:
This is a possible refactor: (including semicolons)
```
var retrieveOrders = function () {
  orders = [];
  fetch(url).then(function (data) {
    data.json().then(function (data) {
      for (key in data) {
        orders.push(data[key]);
      })
      displayOrders();
    });
  });
};
```

Reply:
Thank you for your suggestion! This really helped to clean up and shorten my retrieveOrders function! 🙂